examples/with-angular/app.component.ts

```diff
@@ -9,17 +11,18 @@ const _ud = userdive()
   template: `<router-outlet></router-outlet>`
 })
 export class AppComponent {
-  constructor (@Inject(Router) private router: Router) {}
+  constructor (@Inject(Router) private router: Router) {
+    this.router.events
+    .filter((e) => (e instanceof NavigationEnd))
+    .pairwise()
+    .subscribe(() => {
+      const w = window as any
+      w._ud('send', 'pageview')
```

Comment:
`_ud` object constructed inner angular app as command queue.
Invoked all queue commands, the namespace will swap global function.
So, handler should refer global `_ud` object!